src/components/constant.js

```diff
@@ -0,0 +1 @@
+export default 'http://localhost:8080';
```

Comment:
Why didn't you name the constant? How then in another file to understand what you used, what kind of constant, what does it contain?